openfact.gemspec

```
@@ -48,6 +48,6 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'base64', '>= 0.1', '< 0.4'
   spec.add_runtime_dependency 'hocon', '~> 1.3'
-  spec.add_runtime_dependency 'logger', '~> 1.7'
+  spec.add_runtime_dependency 'logger', '>= 1.5'
```

```suggestion
  spec.add_runtime_dependency 'logger', '~> 1.5'
```
I guess we want to exlude the next major release, to not break our pipeline by accident?